core-models/src/main/scala/com/mesosphere/usi/core/models/RunTemplateLike.scala

```diff
@@ -0,0 +1,134 @@
+package com.mesosphere.usi.core.models
+
+import com.mesosphere.usi.core.models.resources.ResourceRequirement
+import com.typesafe.scalalogging.StrictLogging
+import org.apache.mesos.v1.{Protos => Mesos}
+
+import scala.collection.JavaConverters._
+
+sealed trait RunTemplateLike {
+  final lazy val allResources: Seq[ResourceRequirement] = {
+    val b = Seq.newBuilder[ResourceRequirement]
+    taskResourceRequirements.values.foreach { r =>
+      r.foreach(b += _)
+    }
+    executorResources.foreach { b += _ }
+    b.result()
+  }
```

Comment:
`allResources = taskResourceRequirements.valueIterator.toVector` should do exactly that.

Reply:
This has been replaced with a Map, and, please see this included the executorResources also